pkg/agent/proxy/proxier.go

```diff
@@ -42,6 +42,8 @@ import (
 const (
 	resyncPeriod  = time.Minute
 	componentName = "antrea-agent-proxy"
+
+	maxEnpoints = 1169
```

Comment:
s/maxEnpoints/maxEndpoints

this needs a comment

Reply:
Maybe let's use a number that is eaiser to understand and remember, to avoid unnecessary questions about the number. For example, if the actual maximum is 1169, then just use 1000?